requirements/test-requirements.txt

```diff
@@ -0,0 +1,43 @@
+alabaster==0.7.12
+attrs==22.1.0
+Babel==2.10.3
+breathe==4.34.0
+certifi==2022.9.24
+charset-normalizer==2.1.1
+colorama==0.4.5
+docutils==0.19
+greenlet==1.1.3
+idna==3.4
+imagesize==1.4.1
+iniconfig==1.1.1
+Jinja2==3.1.2
+livereload==2.6.3
+MarkupSafe==2.1.1
+packaging==21.3
+piccolo-theme==0.14.0
+playwright==1.27.1
+pluggy==1.0.0
+py==1.11.0
+pyee==8.1.0
+Pygments==2.13.0
+pyparsing==3.0.9
+pytest==7.1.3
+pytest-base-url==2.0.0
+pytest-playwright==0.3.0
+python-slugify==6.1.2
+pytz==2022.4
+requests==2.28.1
+six==1.16.0
+snowballstemmer==2.2.0
+Sphinx==5.2.3
+sphinx-autobuild==2021.3.14
+sphinxcontrib-applehelp==1.0.2
+sphinxcontrib-devhelp==1.0.2
+sphinxcontrib-htmlhelp==2.0.0
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.3
+sphinxcontrib-serializinghtml==1.1.5
+text-unidecode==1.3
+tomli==2.0.1
+tornado==6.2
+urllib3==1.26.12
```

Comment:
I'd be tempted to simplify this list a bit. I assume you did `pip freeze`, which is OK, but we should be able to get away with just these:
```playwright==1.27.1
pytest==7.1.3
pytest-playwright==0.3.0
Sphinx==5.2.3
sphinx-autobuild==2021.3.14
```

Reply:
Yea. This is what I did. I wasn't sure whether this was right or not. Cheers for the feedback!